src/javaharness/java/arcs/api/PortableJson.java

```diff
@@ -0,0 +1,35 @@
+package arcs.api;
+
+/**
+ * Portable representation of JSON structure.
+ */
+public interface PortableJson {
```

Comment:
Is this and the parser new/original code or brought in from somewhere existing?

Reply:
This is a new interface I provide. It will allow the harness code to parse JSON and not have to directly reference either Android's org.json package or JS APIs.  It's a very thin utility class that is backed by platform dependent parser. This allows the api/ dir to contain platform independent code.